tests-stress/taurus/run-jmeter.ps1

```diff
@@ -0,0 +1 @@
+jmeter -n -t '.\Test Send Notification.jmx' -o test-output -j test.log -q config.properties
```

Comment:
Remove spaces in the filename? Can we run this on UNIX systems?

Reply:
Nope, I can add a sh file.